rootfs/opt/kubectl/configmap/init/mariadb-configure.sh

```diff
@@ -11,6 +11,21 @@
 set -e
 set -x
 
+# Require root privileges
+##
+MARIADB_CONF="/config-etc/my.cnf"
+if [ -e "$MARIADB_CONF" ]; then
+    mkdir -p /etc/mysql
+    ln -sf "$MARIADB_CONF" /etc/mysql/my.cnf
+fi
+
+if ! id 1000 > /dev/null 2>&1
```

Comment:
I've seen the hardwired UIDs for user `qserv` in many files of this package. Could they be pushed into some configuration file instead? That file could set some environment variable `QSERV_UID=1000` which could be used everywhere. This may help with deployments where (for some reason) 1000 can't be used.